server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -115,7 +119,7 @@ export default class BulkChangeModal extends React.PureComponent {
     return this.props.fetchIssues({ additionalFields: 'actions,transitions', ps: 250 });
   };
 
-  handleAssigneeSearch = (query: string) => {
+  handleAssigneeSearch = (query /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)